XiEditor/Search.swift

```diff
@@ -21,13 +21,169 @@ extension NSColor {
 }
 
 class FindViewController: NSViewController, NSSearchFieldDelegate, NSControlTextEditingDelegate {
-    weak var findDelegate: FindDelegate!
+    var findDelegate: FindDelegate!
```

Comment:
be aware of the problem addressed in #235; if this introduces a reference cycle we can end up leaking memory. cc @ryanbloom, thoughts?

Reply:
@cmyr The leak isn't happening on master. In this PR, I can find 5 classes that have suspicious instance counts (even after making `findDelegate` weak):

* `FindSearchField`
* `FindViewController`
* `SupplementaryFindViewController`
* `Document`
* `Label`

So I guess that's a place to start.

Reply:
The fixes in https://github.com/xi-editor/xi-mac/pull/235 probably got lost during some rebase :/ I added them again. How can I get the number of instances to check whether the leak is fixed?

Reply:
@scholtzan If you activate the [Xcode memory debugger](https://useyourloaf.com/blog/xcode-visual-memory-debugger/) the instance counts show up in the sidebar.